### PR TITLE
chore: remove behavior

### LINKED ISSUE #29: Move behaviors to behavior module
__What should we do?__

Behaviors in this module are legacy and we have a [dedicated behavior module](https://github.com/camunda/camunda-bpmn-js-behaviors) to host all sorts of modeling behaviors.

We should migrate what is left in this module over.

__Why should we do it?__

Have a single place, not multiple places, to host behaviors.

### PR DESCRIPTION
**Pull request by philippfromme**
BREAKING CHANGES

* behavior moved to https://github.com/camunda/camunda-bpmn-js-behaviors

---

Closes #29
Related to https://github.com/camunda/camunda-bpmn-js-behaviors/pull/5

### CODE REVIEW

**Review by nikku — CHANGES_REQUESTED** [review]
What's missing here:

* Update to README (pointing to camunda-bpmn-js-behaviors or something).

**Comment by philippfromme** [comment]
> * Update to README (pointing to camunda-bpmn-js-behaviors or something).

Done: https://github.com/camunda/zeebe-bpmn-moddle/pull/33/commits/9f558bcf3c258f0f14e9728cfba418ccefbe56a4

File: README.md [hunk 1]
```diff
@@ -39,26 +39,9 @@ Perform a complete build of the application via
...
-      zeebeExtensionModule
-    ],
-    moddleExtensions: {
-      zeebe: zeebeModdle
-    }
-  });
-```
-
-This extension hooks into the copy mechanism provided by the BPMN editor and ensures Zeebe properties are kept and or dropped on copy and element replace.
+This extension should be used with [camunda-bpmn-js-behaviors](https://github.com/camunda/camunda-bpmn-js-behaviors) to ensure Camunda properties are created, updated and deleted as expected.
```
Comment (nikku) [inline README.md:44]: Could deep link camunda cloud extension section here.

Comment (philippfromme) [reply README.md:44]: Good suggestion. 👍🏻

Comment (philippfromme) [reply README.md]: Done (cf. https://github.com/camunda/zeebe-bpmn-moddle/pull/33/commits/89849e90528316b0d6ac04ab4d54529ce5bbaf7a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R44).

**Review by nikku — APPROVED** [review]

**Pull request merged**